File: VideoStreamerUI/src/app/entities/video-chat/connectToRoom.ts
import { VideoInfo } from '../videoInfo';
import { SocialUser } from 'angularx-social-login';

export class ConnectToRoom {
    SocketId: string;
    VideoInfo: VideoInfo
    User: SocialUser;

    constructor(socketId: string, videoInfo: VideoInfo, user: SocialUser) {
        this.SocketId = socketId;
        this.VideoInfo = videoInfo;
        this.User = user;
    }
}

File: StreamProviderWS/StreamProviderWS/Models/WebSocket/RequestsMessages/Video/WebRtc/ConnectToRoom.cs
using StreamProviderWS.Models.Common;

namespace StreamProviderWS.Models.WebSocket.RequestsMessages
{
    public class ConnectToRoom
    {
        public string SocketId { get; set; }
        public VideoInfo VideoInfo { get; set; }
        public User User { get; set; }
    }
}
